apps/stack/src/extract/extract-namespace-members.ts

```diff
@@ -90,7 +90,7 @@ const extractNamespaceMembersPage = async ({ namespace, repositoryId, sourceCont
     externalNamespaceId: namespace.externalId,
     namespaceName: namespace.name,
     repositoryId,
-    perPage: perPage,
+    perPage: perPage || Number(Config.PER_PAGE),
```

Comment:
Is there a reason for

perPage: perPage || Number(Config.PER_PAGE),

and not just 

perPage: Number(Config.PER_PAGE)

like in the extract-merge-requests.ts file?

Reply:
With new commit refactored to be consistent